.circleci/config.yml

```diff
@@ -158,8 +158,8 @@ jobs:
             - /tmp/ds005/derivatives/freesurfer
 
   test_pytest:
-    machine:
-      image: circleci/classic:201711-01
+    docker:
+      - image: circleci/python:latest
```

Comment:
Why not pin a version (e.g. python:3.7.4)?

Reply:
fmriprep build docs is using latest. Want me to pin it?

Reply:
I think pinning will preempt having sudden errored builds, it is a good idea IMHO